catalog/spatial/registry/registry-common/src/main/java/org/codice/ddf/registry/common/RegistryConstants.java

```diff
@@ -16,6 +16,8 @@
 public class RegistryConstants {
     public static final String REGISTRY_TAG = "registry";
 
+    public static final String REGISTRY_TAG_INTERNAL = "registry.internal";
```

Comment:
This is not an attribute name but a value so i don't think we want to have the namespace on it.